docs/docs/Components/components-logic.md

```diff
@@ -1,3 +1,8 @@
+---
+title: Logic
```

Comment:
What about logic? Update the title to make it more descriptive.

Reply:
Title is below